README.md

```diff
@@ -28,57 +33,102 @@ Project DeepSpeech is an open source Speech-To-Text engine. It uses a model trai
 * [pandas](https://pypi.python.org/pypi/pandas)
 * [DeepSpeech native client libraries](https://tools.taskcluster.net/index/artifacts/#project.deepspeech.deepspeech.native_client.master/)
 
-## Installation
+## Getting the code
 
-### Installing pre-built DeepSpeech Python bindings
+Manually install [Git Large File Storage](https://git-lfs.github.com/), then clone the repository normally:
 
-Pre-built binaries can be found on TaskCluster. You'll need to download `native_client.tar.xz` and the appropriate Python wheel package.
+```bash
+git clone https://github.com/mozilla/DeepSpeech
+```
 
-[native_client.tar.xz (Linux / amd64)](https://index.taskcluster.net/v1/task/project.deepspeech.deepspeech.native_client.master.cpu/artifacts/public/native_client.tar.xz)
-[deepspeech-0.0.1-cp27-cp27mu-linux_x86_64.whl (Linux / amd64)](https://index.taskcluster.net/v1/task/project.deepspeech.deepspeech.native_client.master.cpu/artifacts/public/deepspeech-0.0.1-cp27-cp27mu-linux_x86_64.whl)
-[Other configurations](https://tools.taskcluster.net/index/artifacts/#project.deepspeech.deepspeech.native_client.master/project.deepspeech.deepspeech.native_client.master)
+## Installing a client
+
+If all you want to do is use an already trained model for inference, you can grab one of the pre-built binaries from TaskCluster. You can use a command-line binary, a Python package, or a Node.JS package.
```

Comment:
Does it hinder their understanding to know this name? Would it be any different if it said "you can grab one of the binaries from our GitHub Releases page"?

Reply:
I'd say using "GitHub Releases page" is different from "TaskCluster".

An average developer today will know what "GitHub" is. Only Mozillians will know what "TaskCluster" is. So an average developer reading this will either completely ignore it, all the while hoping they can get away with ignoring it, or spend wasted time trying to figure out what "TaskCluster".

I don't see the benefit of including it, maybe I'm missing something. If there is some benefit, could you tell me why?

Reply:
The benefit I see is that it establishes the name of the place where they're hosted, so that running `util/taskcluster.py` makes more sense. I don't think it's a big deal, so I'll remove the reference here.